### ISSUE TITLE
I needed to create view controller such as application in App Store include another view under camera view that to oriented user front and back of ID 

### ISSUE THREAD

**Issue by AbdoElabd** [BlinkID/blinkid-ios#212]
### [_**###**_](url) [READ] Step 1: Are you in the right place?

  * For general technical questions, post a question on [StackOverflow](http://stackoverflow.com/)
    with BlinkID tag.
  * For issues or feature requests related to __the code in this repository__
    file a Github issue.
    * If this is a __feature request__ make sure the issue title starts with "FR:".
  * For help troubleshooting your application that does not fall under one
    of the above categories, reach out to the personalized
    [Microblink support](https://help.microblink.com/).

### [REQUIRED] Step 2: Describe your environment

  * Xcode version: _____
  * BlinkID SDK version: _____
  * What are you trying to scan? _____

#### Steps to reproduce:

What happened? How can we make the problem occur?
This could be a description, log/console output, etc.

#### Relevant Code:

```
// TODO(you): code here to reproduce the problem
```

**Comment by jcular** [comment]
Hi @AbdoElabd,

BlinkID app on the App Store uses custom overlay view controller, this overlay is not available in the SDK, but a similar overlay is included in the SDK, [MBDocumentVerificationOverlayViewController](http://blinkid.github.io/blinkid-ios/Classes/MBDocumentVerificationOverlayViewController.html).

Here's a modified view controller from our Sample:

```
class ViewController: UIViewController {
    
    var mrtdRecognizer : MBMrtdCombinedRecognizer?
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Valid until: 2018-12-24
        MBMicroblinkSDK.sharedInstance().setLicenseResource("blinkid-license", withExtension: "txt", inSubdirectory: "", for: Bundle.main)
    }

    @IBAction func didTapScan(_ sender: AnyObject) {
        
        // To specify we want to perform MRTD (machine readable travel document) recognition, initialize the MRTD recognizer settings
        self.mrtdRecognizer = MBMrtdCombinedRecognizer()
        self.mrtdRecognizer?.returnFullDocumentImage = true;
        self.mrtdRecognizer?.allowUnverifiedResults = true;
      
        let settings : MBDocumentVerificationOverlaySettings = MBDocumentVerificationOverlaySettings()
        
        /** Crate recognizer collection */
        let recognizerList = [self.mrtdRecognizer!]
        let recognizerCollection : MBRecognizerCollection = MBRecognizerCollection(recognizers: recognizerList)
        
        /** Create your overlay view controller */
        let overlayViewController : MBDocumentVerificationOverlayController = MBDocumentVerificationOverlayViewController(settings: settings, recognizerCollection: recognizerCollection, delegate: self)
        
        /** Create recognizer view controller with wanted overlay view controller */
        let recognizerRunneViewController : UIViewController = MBViewControllerFactory.recognizerRunnerViewController(withOverlayViewController: overlayViewController)
        
        /** Present the recognizer runner view controller. You can use other presentation methods as well (instead of presentViewController) */
        self.present(recognizerRunneViewController, animated: true, completion: nil)
    }
}

extension ViewController: MBDocumentVerificationOverlayViewControllerDelegate {
    
    func documentVerificationOverlayViewControllerDidFinishScanning(_ documentVerificationOverlayViewController: MBDocumentVerificationOverlayViewController, state: MBRecognizerResultState) {
        /** This is done on background thread */
        documentVerificationOverlayViewController.recognizerRunnerViewController?.pauseScanning()
        
        var message: String = ""
        var title: String = ""
        
        if (self.mrtdRecognizer?.result.resultState == MBRecognizerResultState.valid) {
            title = "MRTD"
            
            let fullDocumentImage: UIImage! = self.mrtdRecognizer?.result.fullDocumentImage?.image
            print("Got MRTD image with width: \(fullDocumentImage.size.width), height: \(fullDocumentImage.size.height)")
            
            // Save the string representation of the code
            message = self.mrtdRecognizer!.result.description
        }
        
        /** Needs to be called on main thread beacuse everything prior is on background thread */
        DispatchQueue.main.async {
            // present the alert view with scanned results
            
            let alertController: UIAlertController = UIAlertController.init(title: title, message: message, preferredStyle: UIAlertControllerStyle.alert)
            
            let okAction: UIAlertAction = UIAlertAction.init(title: "OK", style: UIAlertActionStyle.default,
                                                             handler: { (action) -> Void in
                                                                self.dismiss(animated: true, completion: nil)
            })
            alertController.addAction(okAction)
            documentVerificationOverlayViewController.present(alertController, animated: true, completion: nil)
        }
    }
    
    func documentVerificationOverlayViewControllerDidTapClose(_ documentVerificationOverlayViewController: MBDocumentVerificationOverlayViewController) {
        self.dismiss(animated: true, completion: nil)
    }
}
```

**Comment by AbdoElabd** [comment]
i needed to transfer another ViewController instead of  show alert
controller , application failed to transfer view Controller .

*Regards,*
*Abdelrahman Elabd*

*Senior iOS Developer.*


*T.*    + (20) 01060615066
*E.    **e <hend.omar@its.ws>ngabdoelabd@gmail.com <ngabdoelabd@gmail.com>*

**Comment by jcular** [comment]
Closed because it's a duplicate of #214
